lib/hypervisor/hv_kvm/monitor.py

```diff
@@ -664,7 +626,7 @@ def GetFreePCISlot(self):
 
     """
     slots = bitarray(constants.QEMU_PCI_SLOTS)
-    slots.setall(False) # pylint: disable=E1101
+    slots.setall(False)  # pylint: disable=E1101
```

Comment:
extra whitespace

Reply:
pycharm has reported this

Reply:
PEP 8: E261 at least two spaces before inline comment
https://peps.python.org/pep-0008/#inline-comments

Reply:
Acceptable argument.